notebooks/405-paddle-ocr-webcam/405-paddle-ocr-webcam.ipynb

```diff
@@ -191,8 +191,7 @@
     "Input to text recognition model refers to detected bounding boxes with different image sizes, for example, dynamic input shapes. Hence:\n",
     "\n",
     "1. Input dimension with dynamic input shapes needs to be specified before loading text recognition model.\n",
-    "2. Dynamic shape is specified by assigning -1 to the input dimension or by setting the upper bound of the input dimension using, for example, `Dimension(1, 512)`.",
-    "Note: Since the text recognition model is with dynamic input shape and current release of OpenVINO 2022.1 does not support dynamic shape on iGPU, you cannot directly switch device to iGPU for inference in this case. Otherwise, you may need to resize the input images to this model into a fixed size and then try running the inference on iGPU."
+    "2. Dynamic shape is specified by assigning -1 to the input dimension or by setting the upper bound of the input dimension using, for example, `Dimension(1, 512)`.Note: Since the text recognition model is with dynamic input shape and current release of OpenVINO 2022.2 does not support dynamic shape on iGPU, you cannot directly switch device to iGPU for inference in this case. Otherwise, you may need to resize the input images to this model into a fixed size and then try running the inference on iGPU."
```

Comment:
```suggestion
    "2. Dynamic shape is specified by assigning -1 to the input dimension or by setting the upper bound of the input dimension using, for example, `Dimension(1, 512)`.",
    "Note: Since the text recognition model is with dynamic input shape and the current release of OpenVINO 2022.2 does not support dynamic shape on iGPU, you cannot directly switch the device to iGPU for inference in this case. Otherwise, you may need to resize the input images to this model into a fixed size and then try running the inference on iGPU."
```

Reply:
updated